geode-gfsh/src/main/java/org/apache/geode/management/internal/cli/commands/ConnectCommand.java

```diff
@@ -124,6 +126,14 @@ public ResultModel connect(
       useSsl = true;
     }
 
+    if ("".equals(token)) {
+      return ResultModel.createError("--token requires a value, for example --token=foo");
+    }
+
```

Comment:
Like we discussed in the meeting, maybe checks for empty strings for username and password here as well, so that you don't need to deal with them anymore.

Reply:
I think the current code on this branch is correct. We were very careful to not change the existing behavior of --user and --password in order to be backwards compatible. But we also wanted to be able to tell if both --token and --user or --password were used together. That is why we added a specified default of "" for --user and --password. But the old behavior if you specified --user with no value was to just ignore it (it was not flagged as an error). If we do want to say this old behavior is a bug that needs to be fixed lets do that with its own ticket instead of trying to add it into this PR that is about adding --token support.